src/main/java/emissary/server/EmissaryServer.java

```diff
@@ -595,14 +591,6 @@ private ContextHandler buildEmissaryHandler() throws EmissaryException {
         return emissaryHolderContext;
     }
 
-    private ContextHandler buildLogbackConfigHandler() {
```

Comment:
Do we have an alternate mechanism for changing log levels in place? Is this just removal of unused code, or do we have a replacement for this or do we net lose functionality there?

Reply:
This will require further investigation to find a Java 11 alternative for ```ViewStatusMessagesServlet```